dockerfiles/base/scripts/base/commands/cmd_start.sh

```diff
@@ -87,28 +87,30 @@ cmd_start_check_host_resources() {
   HOST_RAM=$(echo ${HOST_RAM#*:} | xargs)
   HOST_RAM=${HOST_RAM% *}
 
+  COMPARE=$(echo "$HOST_RAM<$CHE_MIN_RAM" | bc -l)
```

Comment:
we could use awk so we don't need to add another tool inside the image

/ # awk 'BEGIN { print (1.543 >= 5.234) ? 0  : 1}'
1
/ # awk 'BEGIN { print (15.43 >= 5.234) ? 0  : 1}'
0